rag/advanced_rag/harness/tools/search.py

```
@@ -277,6 +280,288 @@ async def bm25_search(tools, query: str, kb_ids: list[str] | None = None, top_n:
+        "docnm_kwd",
+        "from_entity_kwd",
+        "to_entity_kwd",
+        "name_kwd",
+    ]
+    exprs = []
+    if text:
+        embd_mdl = getattr(tools, "embed_mdl", None)
+        if embd_mdl:
+            try:
+                exprs.append(await settings.retriever.get_vector(text, embd_mdl, top_n, 0.1))
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)